pkg/snapshot/filesystem.go

```diff
@@ -817,7 +849,59 @@ func getFileSystemResetWarningMsg(fileSystemConfig types.FileSystemConfig) strin
 	return fmt.Sprintf("The %s directory was previously configured by a different minio instance.\nProceeding will re-configure it to be used only by the minio instance we deploy to configure the file system, and any other minio instance using this location will no longer have access.\nIf you are attempting to fully restore a prior installation, such as a disaster recovery scenario, this action is expected.", path)
 }
 
-func GetCurrentFileSystemConfig(ctx context.Context, namespace string) (*types.FileSystemConfig, error) {
+func GetCurrentFileSystemConfig(ctx context.Context, namespace string, isMinioDisabled bool) (*types.FileSystemConfig, error) {
+
```

Comment:
would love to keep the current pattern of not having the function begin with a new line :)